README.md

```diff
@@ -561,13 +561,12 @@ $ composer install --dev
 Then, run the following command:
 
 ```
-$ phpunit
+$ ./vendor/bin/phpunit
 ```
 
 You'll obtain some _skipped_ unit tests due to the need of API keys.
 
-Rename the `phpunit.xml.dist` file to `phpunit.xml`, then uncomment the
```

Comment:
One should rename the phpunit.xml.dist file to phpunit.xml to avoid accidentally checking in your API keys.